src/backend/tools/py.CLI/caliopen_cli/cli.py

```diff
@@ -58,6 +58,11 @@ def main(args=sys.argv):
     sp_shell = subparsers.add_parser('shell')
     sp_shell.set_defaults(func=shell)
 
+    sp_set_ttls = subparsers.add_parser('set_notifs_ttls',
```

Comment:
need update README.md documentation to not forget to use this command during development setup

Reply:
commit #911cf89